migration/mule/scatter-gather-flow-control/ScatterGatherFlowControl/src/main/synapse-config/sequences/aggregate_seq.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<sequence name="aggregate_seq" trace="disable" xmlns="http://ws.apache.org/ns/synapse">
+    <property name="aggregated" scope="default">
+        <aggregated xmlns=""/>
+    </property>
+    <aggregate id="aggregate_id">
+        <completeCondition>
+            <messageCount max="-1" min="-1"/>
```

Comment:
Does aggregate mediator detect that it has to wait for 2 clones?

Reply:
Fixed.